content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -125,31 +63,46 @@ The following example configures the Istio CNI plugin to ignore pods in the name
 | `cniConfDir` | | `/etc/cni/net.d` | Must be the same as the environment's `--cni-conf-dir` setting (`kubelet` parameter). |
 | `cniConfFileName` | | None | Leave unset to auto-find the first file in the `cni-conf-dir` (as `kubelet` does).  Primarily used for testing `install-cni` plugin configuration.  If set, `install-cni` will inject the plugin configuration into this file in the `cni-conf-dir`. |
 
-### Hosted Kubernetes Usage
+### Excluding specific Kubernetes namespaces
+
+This example configures the Istio CNI plugin via `helm template` to modify the log level and
+ignore pods in the namespaces `istio-system`, `foo_ns`, and `bar_ns`.
+Refer to the [Customizable Install with Helm](/docs/setup/kubernetes/install/helm/#cni) for complete instructions.
+
+Render and apply Istio CNI components and override the default configuration of the `istio-cni` Helm
+chart's `logLevel` and `excludeNamespaces` parameters:
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio-cni --name=istio-cni --namespace=istio-system \
+    --set logLevel=info \
+    --set excludeNamespaces={"istio-system,kube-system,foo_ns,bar_ns"} | kubectl apply -f -
+{{< /text >}}
+
+### Hosted Kubernetes settings
 
-Not all hosted Kubernetes clusters are created with the `kubelet` configured to use the CNI plugin so
-compatibility with this `istio-cni` solution is not ubiquitous.  The `istio-cni` plugin is expected
-to work with any hosted Kubernetes leveraging CNI plugins.  The below table indicates the known CNI status
-of many common Kubernetes environments.
+The Istio CNI solution is not ubiquitous. Some platforms, especially hosted Kubernetes environments, do not enable the CNI plugin in the kubelet configuration.
```

Comment:
Please respect the 80 character length.
```suggestion
The Istio CNI solution is not ubiquitous. Some platforms, especially hosted Kubernetes environments,
do not enable the CNI plugin in the kubelet configuration.
```

Reply:
That would result in a lot of churn in this file. It seems like it's been edited with ~100 columns. I'll stick to that and fix the lines longer than 100 columns.